packages/ckeditor5-engine/src/conversion/upcasthelpers.ts

```diff
@@ -1072,7 +1071,7 @@ function prepareToAttributeConverter(
 			config.model.value( data.viewItem, conversionApi, data ) : config.model.value;
 
 		// Do not convert if attribute building function returned falsy value.
-		if ( modelValue === null ) {
+		if ( modelValue === null || modelValue === undefined ) {
```

Comment:
Without the check for `undefined`, the following code attempted to apply a model attribute with an `undefined` value (with no effect, as the model writer ignores changes from `undefined` to `undefined`). It consumed it without applying anything to the model. The issue is that we often use conversion helpers simply by putting `return` without any value, and TypeScript is not helping here either.